lib/rbUtil.js

```diff
@@ -40,6 +40,38 @@ rbUtil.parseURL = function parseURL (uri) {
     }
 };
 
+/**
+ * Replaces subdomain with a wildcard
+ *
+ * @param domain a full domain name (e.g. en.wikipedia.org)
+ * @returns {string} wildcard version (e.g. *.wikipedia.org)
+ */
+rbUtil.wildcardSubdomain = function(domain) {
+    if ((domain.match(/\./g) || []).length >= 2) {
+        return '*.' + domain.replace(/^[^.]+\./, "");
+    } else {
+        return domain;
+    }
+};
+
+/**
+ * Constructs Content-Security-Policy header to send in response
+ *
+ * @param allowInline if true 'unsafe-inline' is added to style-src
+ * @returns {string} CSP header value
+ */
+rbUtil.constructCSP = function(req, allowInline) {
```

Comment:
Minor nit: It might be slightly cleaner to only pass in the domain, instead of the entire req. This will keep this utility function independent of the req structure, and make it clearer which part of the request is actually used.

Additionally, it might be clearer to accept an options object with an `allowInline` field. The advantage is that call sites are somewhat self-documenting that way, which is better than a boolean arg.